config/dev.exs

```diff
@@ -70,7 +70,7 @@ config :ret, Ret.Repo,
   username: "postgres",
   password: "postgres",
   database: "ret_dev",
-  hostname: "localhost",
+  hostname: "#{System.get_env("DB_HOST")}" || "localhost",
```

Comment:
this is probably OK, but worth noting that when running this as an erlang release, it'll burn in the environment variable here I think since this is going to be compiled.

Reply:
(since this is `dev` its basically inconsequential)